java/rocksjni/options.cc

```diff
@@ -3840,6 +3840,29 @@ jlong Java_org_rocksdb_Options_blobCompactionReadaheadSize(JNIEnv*, jobject,
   return static_cast<jlong>(opts->blob_compaction_readahead_size);
 }
 
+/*
+ * Class:     org_rocksdb_Options
+ * Method:    setBlobFileStartingLevel
+ * Signature: (JJ)V
+ */
+void Java_org_rocksdb_Options_setBlobFileStartingLevel(
+    JNIEnv*, jobject, jlong jhandle, jint jblob_file_starting_level) {
+  auto* opts = reinterpret_cast<ROCKSDB_NAMESPACE::Options*>(jhandle);
+  opts->blob_file_starting_level =
+      static_cast<uint32_t>(jblob_file_starting_level);
```

Comment:
We would want to use `static_cast<int>` here as well

Reply:
I haven't found this one. :)